README.md

```diff
@@ -1076,6 +1076,10 @@ The use of Amazon Voice Focus via this SDK involves the downloading and executio
 
 The use of Amazon Voice Focus runtime code is subject to additional notices. See [this NOTICES file](https://static.sdkassets.chime.aws/workers/NOTICES.txt) for details. You agree to make these additional notices available to all end users who use Amazon Voice Focus runtime code via this SDK.
 
+The browser demo applications in the demos directory (https://github.com/aws/amazon-chime-sdk-js/tree/master/demos) use TensorFlow.js (https://github.com/tensorflow/tfjs) and pre-trained TensorFlow.js models (https://github.com/tensorflow/tfjs-models) for image segmentation. Use of these third party models involves downloading and execution of code at runtime from jsDelivr (https://www.jsdelivr.com/) by end user browsers. For the jsDelivr Acceptable Use Policy, please visit this link (https://www.jsdelivr.com/terms/acceptable-use-policy-jsdelivr-net).
```

Comment:
If these are demo specific notifices, can we can move this under demos/browser README?

Reply:
No; this is a legal notice, so we don't want to bury it. It might be worth putting it in both places.

Reply:
Got it!